sections/main-product.liquid

```diff
@@ -773,40 +175,43 @@
               {{ 'products.product.xr_button' | t }}
             </button>
           {%- endif -%}
-          {%- if media_count > 1 and section.settings.gallery_layout contains 'thumbnail' or section.settings.mobile_thumbnails == 'show' -%}
+          {%- if media_count > 1
+            and section.settings.gallery_layout contains 'thumbnail'
+            or section.settings.mobile_thumbnails == 'show'
+          -%}
             <slider-component
-              id="GalleryThumbnails-{{ section.id }}-right"
+              id="GalleryThumbnails-{{ section.id }}"
               class="thumbnail-slider slider-mobile-gutter quick-add-hidden{% unless section.settings.gallery_layout contains 'thumbnail' %} medium-hide large-up-hide{% endunless %}{% if section.settings.mobile_thumbnails != 'show' %} small-hide{% endif %}{% if media_count <= 3 %} thumbnail-slider--no-slide{% endif %}"
```

Comment:
Here too, I wonder if that logic should be removed?  Should not be something being touched in this PR i believe.